contents/marketplace/guidelines.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Marketplace Guidelines
+showTitle: true
+hideLastUpdated: true
+---
+
+## Getting listed
+
+Please [submit a PR to posthog.com](https://github.com/PostHog/posthog.com). You'll need to:
+
+* Take our [listing template](listing-template), and create a copy in posthog.com/contents/marketplace/YOUR-NAME.md
+* PostHog's team will then review your pull request. We will:
+  * Add a contact form
+  * Add you to our menu structure
+  * Clarify if we've any questions
+  
+## Why do this
+
+PostHog is used by tens of thousands of users at thousands of companies, interested in improving their products.
+
+They range from hobbyists and startups, to large enterprises.
+
+We frequently have users asking for support in managing their instance, for customization or for features we don't provide.
+
+Being listed in the PostHog Marketplace is a way of you having these users discover your PostHog-based product or service on the internet.
+
+This section of our website is new - our current goal is to promote it more broadly once the first few listings are successful.
+
+## Brand
+
+Our main aim is that it's clear if your product/service is built on PostHog, but that your users are clear that it's not _provided_ by PostHog!
+
+Do these things:
+
+* Use the PostHog name or logo to link to PostHog
```

Comment:
I'd link to our brand assets page and ask people to use these: https://posthog.com/handbook/company/brand-assets